pilot/cmd/pilot-agent/main.go

```diff
@@ -373,9 +373,7 @@ var (
 				log.Info("JWT policy is first-party-jwt")
 				jwtPath = securityModel.K8sSAJwtFileName
 			} else {
-				err := fmt.Errorf("invalid JWT policy %v", jwtPolicy.Get())
-				log.Errorf("%v", err)
-				return err
+				log.Info("Using existing certs")
```

Comment:
- Can you add more comments to the code on how the authentication using the existing certs work?
- Is the authentication using the existing cert based on the possession of an existing non-expired public cert? Or is the authentication based on the possession of an existing non-expired public cert and the private key? If it is the latter, where is the code authenticating the possession of the private key? If it is the former, does the possession of a public key certificate (which is not a secret) result in the presenter always being able to get a new certificate?

Reply:
Yes, that's how VMs have worked since 0.2 when we launched them.

The authentication code was in citadel - this PR is adding it to Istiod ( the extra authenticator ).  

The auth is using same mTLS we use for workload authentication - it depends on user provisioning certificates using some tool ( puppet, ansible, script, etc ), which is common for 
managing VMs.

Reply:
Yes, that's how VMs have worked since 0.2 when we launched them.

The authentication code was in citadel - this PR is adding it to Istiod ( the extra authenticator ).  

The auth is using same mTLS we use for workload authentication - it depends on user provisioning certificates using some tool ( puppet, ansible, script, etc ), which is common for 
managing VMs.